ansible_collections/arista/avd/roles/eos_validate_state/ANTA-Preview.md

```diff
@@ -15,7 +15,7 @@ title: Ansible Collection Role eos_valudate_state - Preview Integration with ANT
     If you have any questions, please leverage the GitHub [discussions board](https://github.com/aristanetworks/avd/discussions)
 
 !!! warning
-    ANTA version has been bumped to **0.12.0**. Please make sure you are running this exact version! For more details, please see the [installation section](#how-to-run-eos_validate_state-in-anta-mode).
+    ANTA version has been bumped to **0.13.0**. Please make sure you are running this exact version! For more details, please see the [installation section](#how-to-run-eos_validate_state-in-anta-mode).
```

Comment:
Can you fix the typo line3 of this file `valudate` :)

Reply:
Fixed in 183de5840e99bdad7fdf14a5aea6e20aa8016765